### PR TITLE
Fixing two issues

### PR DESCRIPTION
**Pull request by RaananW**
See https://forum.babylonjs.com/t/malformed-regular-expression-and-reassignment-issue-in-inspector/46269

### CODE REVIEW

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/14607/merge

Test environment: 
https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14607/merge/index.html

To test a playground add it to the URL, for example:

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14607/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/14607/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/14607/merge
https://gui.babylonjs.com/?snapshot=refs/pull/14607/merge
https://nme.babylonjs.com/?snapshot=refs/pull/14607/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/14607/merge#BCU1XR#0

File: packages/dev/sharedUiComponents/src/stringTools.ts [hunk 1]
```diff
@@ -46,7 +46,7 @@ export class StringTools {
                 if (popup) {
                     popup.location.href = url;
                 } else {
-                    location = url;
+                    location.href = url;
```
Comment (sebavan) [inline packages/dev/sharedUiComponents/src/stringTools.ts:49]: in this case should we not update 47 as well ?

Comment (RaananW) [reply packages/dev/sharedUiComponents/src/stringTools.ts:49]: Nothing is wrong in 47. Or am i missing something?

Comment (sebavan) [reply packages/dev/sharedUiComponents/src/stringTools.ts:49]: ahah I was reading a removed href instead of added.... stupid me...

**Review by sebavan — APPROVED** [review]

**Pull request merged**